README.md

```diff
@@ -1,8 +1,52 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nazanin Akbari . Jarvis Consulting
 
+My bachelor degree is in the field of Biomedical engineering. I have two years of work experience as a biomedical engineer in 5Azar hospital, Gorgan, Iran. I started my master degree at Concordia university, Montreal, QC in 2019. I got my master degree from Concordia university in the field of Electrical Engineering and Computer Science. At Concordia university, I did multiple projects in signal processing, image processing, Hardware verification and designing an Airline reservation system and I also designed some games. In Montreal, I also learned French language, B1 level. I like software industry because this field is incredibly broad. I am passionate to learn new things and expand my knowledge and experience. I love to solve problems and help people to experience a better and easier life.
```

Comment:
I started my master degree at Concordia University, Montreal, QC in 2019. I got my master degree from Concordia university in the field of Computer Science and Electrical Engineering . At Concordia University, I did multiple projects in signal processing, image processing, Hardware verification and designing an Airline reservation system and I also designed some games. In Montreal, I also learned French language, B1 level. I like software industry because this field is incredibly broad. I am passionate to learn new things and expand my knowledge and experience. I love to solve problems and help people to experience a better and easier life.

https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee